.idea/runConfigurations/_Generate_FIR_Checker_Boilerplate.xml

```diff
@@ -0,0 +1,24 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name=" Generate FIR Checker Boilerplate" type="GradleRunConfiguration" factoryName="Gradle">
+    <ExternalSystemSettings>
+      <option name="executionName" />
+      <option name="externalProjectPath" value="$PROJECT_DIR$" />
+      <option name="externalSystemIdString" value="GRADLE" />
+      <option name="scriptParameters" value="" />
+      <option name="taskDescriptions">
+        <list />
+      </option>
+      <option name="taskNames">
+        <list>
+          <option value=":compiler:fir:checkers:generateCheckersComponents" />
+          <option value=":idea:idea-frontend-fir:generateCode" />
+        </list>
+      </option>
+      <option name="vmOptions" value="" />
+    </ExternalSystemSettings>
+    <ExternalSystemDebugServerProcess>true</ExternalSystemDebugServerProcess>
+    <ExternalSystemReattachDebugProcess>true</ExternalSystemReattachDebugProcess>
+    <DebugAllEnabled>false</DebugAllEnabled>
+    <method v="2" />
+  </configuration>
+</component>
```

Comment:
Nit: new line :)

Reply:
This is managed by IntelliJ and it never put a newline at the end. Manually putting a newline in the text editor would cause the IDE to remove it when the config is touched again.

Reply:
Ah, too bad. :\ Alright, never mind, then.